plugins/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/vmware-tanzu/antrea/plugins
+
+go 1.13
+
+require (
+	github.com/vmware-tanzu/antrea v0.7.1
```

Comment:
do we need to worry about updating this? Or does it not matter because this module (github.com/vmware-tanzu/antrea/plugins) is only meant to be built by us when we build the `octant-antrea-ubuntu` Docker image, in which case the `replace` directive kicks in and guarantees that we use the latest Antrea code?

Reply:
I guess as long as no one tries to `go get` this module, it's fine...

Reply:
After we check in this change with K8s API 1.18, I think we can use the upstream instead of replace ../

Reply:
It's usually set to v0.0.0 when it's supposed to be replaced.

Reply:
OK, will change it to v0.0.0